augur/refine.py

```diff
@@ -241,11 +241,27 @@ def run(args):
     if args.divergence_units=='mutations-per-site': #default
         pass
     elif args.divergence_units=='mutations':
-        L = tt.seq_len
-        for node in node_data['nodes']:
+        if not args.timetree:
+            tt.infer_ancestral_sequences()
+        nuc_map = profile_maps['nuc']
+
+        def different(a,d):
```

Comment:
Even if we keep this function in augur, can we rename it to something like `are_sequence_states_different` or `sequence_states_are_different`?

Along those same lines, can we use words for variable names here instead of single-letter abbreviations (`a` -> `ancestral` and `d` -> `derived`)?